Makefile

```diff
@@ -243,28 +235,19 @@ uninstall:
 
 uninstall-lite:
 	if test -f nxcomp/Makefile; then ${MAKE} -C nxcomp $@; fi
+	if test -f nxproxy/Makefile; then ${MAKE} -C nxproxy $@; fi
 
-	# uninstall nproxy wrapper script
-	$(RM_FILE) $(DESTDIR)$(BINDIR)/nxproxy
-	# FIXME: don't use uninstall rule in nxproxy/Makefile.in, let's do
-        # it on our own for now...
-	$(RM_FILE) $(DESTDIR)$(NXLIBDIR)/bin/nxproxy
-	$(RM_DIR) $(DESTDIR)$(NXLIBDIR)/bin/
-	$(RM_FILE) $(DESTDIR)$(PREFIX)/share/man/man1/*.1
 	$(RM_FILE) $(DESTDIR)$(PREFIX)/share/nx/VERSION.nxproxy
 	$(RM_DIR) $(DESTDIR)$(PREFIX)/share/nx/
 
 uninstall-full:
-	for f in nxagent; do \
-	    $(RM_FILE) $(DESTDIR)$(BINDIR)/$$f; done
+	if test -f nxcompshad/Makefile; then ${MAKE} -C nxcompshad $@; fi
```

Comment:
`"$@"`

Besides, are the uninstall rules good enough? The problem here especially is `DESTDIR`, which, since we don't add it to the `prefix` any longer, won't be prepended to the path, hence deleting files from the actual file system instead of the `DESTDIR`, like we did before.

Reply:
>The DESTDIR logic should be in the nxproxy, nxcomp and nxcompshad Makefiles, coming from autotools. As it works for our install rules, I suppose it works for uninstall (in theory), too. However, the uninstall rules are mainly for people doing tarball installs directly onto the system.

That seems to be true - looking through the generated Makefiles, I see that `DESTDIR` logic is also applied to the `uninstall` target(s), so I guess we're fine.

Reply:
Still ping on `"$@"` instead.